src/extension/geofence-server/src/test/resources/geofence-default-override.properties

```diff
@@ -0,0 +1,3 @@
+
+geofenceDataSource.url=jdbc:h2:mem:geofence
```

Comment:
Maybe a small comment stating why it's using memory (windows) could help preventing future regressions.